orc8r/tools/fab/dev_utils.py

```diff
@@ -118,12 +117,14 @@ def get_next_available_gateway_id(
     Returns:
         Next available gateway ID in the form gwN
     """
-    # gateways is a dict mapping gw ID to full resource
+    # 'gateways' is a dict mapping gw ID to full resource
```

Comment:
Nit here and below: I'm generally not a fan of putting quotes around variable names, because then it's unclear whether you're referring to a variable or a string. To me it's clear to just leave it as it was, but up to you